### ISSUE TITLE
Associate item measurement units (eg cm) to its parent item (accessibility)

### ISSUE THREAD

**Issue by robeastwood-agency** [aehrc/smart-forms#1641]
**Is your feature request related to a problem? Please describe.**
For the purposes of meeting accessibility requirements, when an item has an associated measurement unit (eg cm or kg), the content is visibly associated to the on-screen field, but not so when a screen reader reads the content. The screen reader does not announce the associated units.

**Describe the solution you'd like**
From an accessibility perspective, it is preferrable to have the measurement unit programmatically associated with its parent field, so that the content is read out immediately when the field receives focus. Accessibility experts suggest the use of the `aria-describedby` attribute to associate unit text with the input field.

**Describe alternatives you've considered**
None

**Additional context**
None
